test/scripts/base-host-check.sh

```diff
@@ -98,6 +98,18 @@ check_ca_cert() {
     fi
 }
 
+check_modularity() {
+    # Verify modules that are enabled on a system, if any.
+    modules_expected=$(jq -rc '.expect_modularity.modules[]' "${config}")
```

Comment:
You could either check for explicitly enabled modules in the BP or filter any packages in `.blueprint.pakages[]` for any that start with `@` and contain `:` ?

```jq
jq -rc '.blueprint.packages[] | select(.name | startswith("@") and contains(":"))' "${config}"
```

AFAIK, we never added a custom field into the manifest gen config, which is then used by a test case. Instead, we always inspect the BP and base the check on that.

Reply:
I've adjusted base-host checks to use the blueprint, instead of a new attribute.